vmdb/app/models/computer_system.rb

```diff
@@ -0,0 +1,6 @@
+class ComputerSystem < ActiveRecord::Base
+  belongs_to :configured_system
```

Comment:
The migration has `belongs_to :managed_entity, :polymorphic => true`, but this is different.